### PR TITLE
Qesap e2e verbosity tests

### PR DESCRIPTION
**Pull request by mpagot**
This PR introduce an improvement in e2e test of the glue script. In particular there are many new tests about the output of the qesap.py and for the first time the true execution of the ansible subcommand 

Ticket : https://jira.suse.com/browse/TEAM-6844

VR is pipeline in PR

### CODE REVIEW

File: scripts/qesap/test/e2e/test.sh [hunk 1]
```diff
@@ -21,43 +21,194 @@ rm -rf "${QESAPROOT}"
...
+rc=$?
+if [ $rc -ne 0 ]; then
+  test_die "rc:$rc in verbose mode there should be some DEBUG"
+fi
+grep -qE "^INFO" "${QESAPROOT}/test_4_ansible_pass_verbose.txt"
+rc=$?
+if [ $rc -ne 0 ]; then
+  test_die "rc:$rc in verbose mode there should be some INFO"
+fi
+grep -E "TASK \[Say hello\]" "${QESAPROOT}/test_4_ansible_pass_verbose.txt" | wc -l
```
Comment (mpagot) [inline scripts/qesap/test/e2e/test.sh:211]: this test is intentionally left uncompleted. The expected value should be 1 but, due to a bug in the glue script that has to be fixed, is now 2

Comment (mpagot) [reply scripts/qesap/test/e2e/test.sh:211]: I'll convert it to a real test in a next PR where I'll also at the same time fix the bug.

File: scripts/qesap/test/e2e/test.sh [hunk 2]
```diff
@@ -21,43 +21,194 @@ rm -rf "${QESAPROOT}"
...
 
+test_step "[test_5.yaml] Change a setting in place"
+# This test try to reproduce the situation in which
+# a user run `qesap.py conf.yaml` a first time
+# then tune and change something in the config.yaml
+# and run the `qesap.py conf.yaml` a second time
+# to have the changes applied in the generated .tfvars or ansible files
+qesap.py --verbose -b ${QESAPROOT} -c test_3.yaml configure || test_die "Should exit with zero rc but was not. rc:$?"
+grep -q lampone "${TEST_TERRAFORM_TFVARS}" || test_die "${TEST_TERRAFORM_TFVARS} generated from test_3.yaml should contain the world lampone"
+# test_5 has everything the same of test_3 except a terraform variable,
```
Comment (mpagot) [inline scripts/qesap/test/e2e/test.sh:70]:
```suggestion
# test_5 has everything the same as test_3 except for a terraform variable,
```

**Review by lpalovsky — APPROVED** [review]
I am giving my 0.7 LGTM which is equal to the level of my competency in this matter :D

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Pull request merged**